### PR TITLE
Adding notebook showing how to get predictions

### PR DESCRIPTION
**Pull request by balikasg**
@lastmansleeping this is what I have come up with until now. The remaining thing is to find a way to apply the prefix 
function.

### CODE REVIEW

**Comment by lastmansleeping** [comment]
@balikasg Can you add a section here as well -> https://ml4ir.readthedocs.io/en/latest/advanced/index.html

**Comment by balikasg** [comment]
@lastmansleeping this is now done. 
I only added the section hoping that things will then update automatically. Let me know if this is not the case.

**Comment by balikasg** [comment]
@lastmansleeping are we good to merge?

**Comment by lastmansleeping** [comment]
@balikasg Only the pydocs section update automatically. The other parts need to be added. Were you able to add a section to the advanced guide?

Also, I don't have time to review it today. Is this urgent? I can take a look early next week.

**Comment by balikasg** [comment]
no problem @lastmansleeping , let's do it this week :-)

File: docs/source/advanced/predicting_with_a_trained_model.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        tfrecord_type=TFRecordTypeKey.EXAMPLE,
+        batch_size=1024,
+        preprocessing_keys_to_fns={'trim_text': trim_text},  # IMPORTANT!
+        file_io=file_io, use_part_files=True,
+        logger=logger
+    )
+```
+will result in queries whose size is 3 bytes (as described in `trim_text`).
+
+For more information on these, [please refer to this notebook](../../../python/notebooks/predicting_with_ml4ir.ipynb).
```
Comment (lastmansleeping) [inline docs/source/advanced/predicting_with_a_trained_model.md:128]: @balikasg This link will be broken. Can you use a github.com... URL perhaps? But change it to point to `master`

**Comment by lastmansleeping** [comment]
@balikasg This looks good to me. But if you want to take a day to clean up the notebook (or make it pretty), you can do it.

File: docs/source/advanced/predicting_with_a_trained_model.md (hunk 1 above)
Comment (balikasg) [reply docs/source/advanced/predicting_with_a_trained_model.md]: yes, done now!

**Review by lastmansleeping — APPROVED** [review]
@balikasg Thanks for adding this. Love the docs.

We should find a better system to keep the notebooks updated. Let's discuss more on that.

**Pull request merged**